src/overlays/effects/ovl_Effect_Ss_Kakera/z_eff_ss_kakera.c

```diff
@@ -8,50 +8,349 @@
 
 #define PARAMS ((EffectSsKakeraInitParams*)initParamsx)
 
-s32 EffectSsKakera_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
+u32 EffectSsKakera_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsKakera_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsKakera_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-#if 0
+void func_8097E130(EffectSs* this, GlobalContext* globalCtx);
+f32 func_8097E368(f32, s32);
+f32 func_8097E384(f32, s32);
+f32 func_8097E3C0(f32, s32);
+f32 func_8097E400(f32, s32);
+
 const EffectSsInit Effect_Ss_Kakera_InitVars = {
     EFFECT_SS_KAKERA,
     EffectSsKakera_Init,
 };
+KakeraColorStruct D_8097EAD8[] = { { 0, { 255, 255, 255 } }, { 0, { 235, 170, 130 } }, { 1, { 210, 190, 170 } } };
```

Comment:
This looks like it would be better formatted one entry/line, which I imagine will happen if you add a comma to the last entry and reformat.